extensions/amp-form/0.1/amp-form.js

```diff
@@ -216,6 +221,29 @@ export class AmpForm {
   }
 
   /**
+   * Returns form data as an object.
+   * @return {!Object}
+   * @private
+   */
+  getFormAsObject_() {
```

Comment:
`FormData` works very well in `POST` case and could be great if we decide to support file uploads for example. I'd rather keep `FormData` for what probably going to be the most used case `POST` for a more robust implementation.